### PR TITLE
Update dbaas-tools.spec

### PR DESCRIPTION
**Pull request by BupycHuk**
[PMM-7](https://jira.percona.com/browse/PMM-7)

Build: https://github.com/Percona-Lab/pmm-submodules/pull/2511

https://github.com/percona-platform/dbaas-controller/pull/282
https://github.com/percona/pmm-doc/pull/796

### CODE REVIEW

**Comment by nikita-b** [comment]
@BupycHuk It's a bad package and won't work without additional changes:
1. We need to change this `sed` because goreleaser config was changed.
https://github.com/percona/pmm-server/blob/03f88561c8d4489e97c48865f0b757723a5eca2d/rhel/SPECS/dbaas-tools.spec#L57
2. https://github.com/percona/pmm-server/blob/03f88561c8d4489e97c48865f0b757723a5eca2d/rhel/SPECS/dbaas-tools.spec#L58
The original package doesn't have `make build` anymore:
https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/Makefile#L70
3. We need to add a changelog.

**Comment by BupycHuk** [comment]
Thanks @nikita-b , updated

**Review by nikita-b — APPROVED** [review]

**Comment by denisok** [comment]
@BupycHuk should you also update CI in DBaaS: https://github.com/percona-platform/dbaas-controller/blob/main/.github/workflows/ci.yml#L267 ?

**Comment by denisok** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-submodules/detail/PR-2511/1/pipeline

`binary=dist/aws-iam-authenticator_linux_amd64_v1/aws-iam-authenticator`

probably path was changed, and copying from different path `install -D -p -m 0755 dist/authenticator_linux_amd64/aws-iam-authenticator`

File: rhel/SPECS/dbaas-tools.spec [hunk 1]
```diff
@@ -68,14 +68,17 @@ cd %{_builddir}/aws-iam-authenticator-%{commit_aws}/src/github.com/kubernetes-si
 install -D -p -m 0755 dist/authenticator_linux_amd64/aws-iam-authenticator %{buildroot}/opt/dbaas-tools/bin/aws-iam-authenticator
```
Comment (denisok) [inline rhel/SPECS/dbaas-tools.spec:68]:
```suggestion
install -D -p -m 0755 dist/aws-iam-authenticator_linux_amd64_v1/aws-iam-authenticator %{buildroot}/opt/dbaas-tools/bin/aws-iam-authenticator
```

**Review by denisok — CHANGES_REQUESTED** [review]

**Comment by denisok** [comment]
> https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-submodules/detail/PR-2511/1/pipeline
> 
> `binary=dist/aws-iam-authenticator_linux_amd64_v1/aws-iam-authenticator`
> 
> probably path was changed, and copying from different path `install -D -p -m 0755 dist/authenticator_linux_amd64/aws-iam-authenticator`

https://github.com/Percona-Lab/pmm-submodules/pull/2511

**Review by denisok — APPROVED** [review]

**Pull request merged**